src/Tools/dotnet-gcdump/CommandLine/CollectCommandHandler.cs

```diff
@@ -137,5 +230,13 @@ private static async Task<int> Collect(CancellationToken ct, IConsole console, i
             {
                 Argument = new Argument<int>(name: "timeout", defaultValue: DefaultTimeout)
             };
+        
+        private static Option ConsoleOutOption() =>
+            new Option(
+                aliases: new[] { "--std-out" },
+                description: "Writes plaintext results into stdout (overrides -o)")
```

Comment:
The only reason I'd want to use `--std-out` is when I don't have access to a Windows PC, and In that case that file does not mean anything to me. I think it would be a waste to create a new file every time I run this command. and IMO cli people would much prefer piping, grepping and if required echo to a file `>` as they wish. wdyt?

Reply:
My thinking here was that if we generate the file as well, we could reprint using the additional verb I mentioned before.  I think your point about just redirecting the text to a file is probably good enough for now though.  @shirhatti any thoughts from a usability standpoint?

Reply:
Now that makes sense actually, it'll be helpful in order to conpute a diff later, I'll do it (+additional verb) tomorrow.

Reply:
Hi @josalem , I've pushed some commits (+ updated the PR description), please let me know what do you think.